arcgis-ios-sdk-samples/Shared resources/Libraries/SVProgressHUD/SVProgressHUD.h

```diff
@@ -118,18 +118,6 @@ typedef void (^SVProgressHUDDismissCompletion)(void);
 
```

Comment:
Since this is a third party source file, I wouldn't modify. Unless and until we no longer use it and it can be removed, we can (and should) regularly update it. We've had to do so in the past. Removing these unused methods would needlessly make updating more complicated. Does that make sense?

Reply:
I didn't realize we still updated it. I removed the methods from there so no one would use them in the future, but I don't suppose we need to enforce that at the code level.